kuma/users/jinja2/users/ban_user_and_cleanup.html

```diff
@@ -0,0 +1,73 @@
+{% extends "base.html" %}
+
+{% block body_attributes %}{% endblock %}
+
+{% block bodyclass %}user ban-user{% endblock %}
+
+{% block title %}{{ _('Ban %(user)s', user=detail_user) }}{% endblock %}
+{% set styles = ('dashboards', 'jquery-ui') %}
+{% set classes = 'compare' %}
+
+{% block site_css %}
+  {{ super() }}
+  {% stylesheet 'users' %}
+  {% stylesheet 'user-banned' %}
+{% endblock %}
+
+{% block content %}
+<section id="content">
+
+{% if not user_banned %}
+  <h1>
+    {{ _('Identify <a href="%(url)s">%(user)s</a> as a spammer?', url=detail_user.get_absolute_url(), user=detail_user) }}
+  </h1>
+{% else %}
+  <h1>
+    {{ _('<a href="%(url)s">%(user)s</a> has been banned.', url=detail_user.get_absolute_url(), user=detail_user) }}
+  </h1>
+  <p>
+  {{ _('Because %(user)s has already been banned, you may review and revert their revisions below.', user=detail_user) }}
+  </p>
+{% endif %}
+
+<div class="wrap">
+
+  <section id="content-main" class="full">
+    <section id="user-head" class="column-container column-container-reverse vcard user-head">
+      <div class="column-10">
+        {% include "users/includes/user_title_info.html" %}
+      </div>
+      <div class="column-2 ">
+        {% with accounts=social_accounts(detail_user) %}
+          {% include "users/includes/user_avatar_and_links.html" %}
+        {% endwith %}
+      </div>
+    </section><!-- /#user-head -->
+
+    <section id="docs-activity" class="user-section">
+        {% include "users/includes/user_revision_history_form.html" %}
+
+        <form id="ban-and-cleanup-form" action="{{ url('users.ban_user_and_cleanup_summary', detail_user.id) }}" method="POST">
+
+          {% csrf_token %}
+          <p>
+            {% if not user_banned or revisions %}
+              <button type="submit" class="button negative ban-link">
+                {% if user_banned and revisions %}
+                  {{ _('Revert Revisions') }}
+                {% else %}
+                  {{ _('Ban User for Spam & Revert Revisions') }}
+                {% endif %}
+              </button>
+              <a href="{{ url('users.ban_user',detail_user.id) }}">{{ _('Ban user for other reasons.') }}</a>
```

Comment:
This line should be moved up into ``{% if user_banned and revisions %}``:

<img width="397" alt="revert or ban" src="https://cloud.githubusercontent.com/assets/286017/16312060/959ec958-3938-11e6-949a-fed0b6f12445.png">